### PR TITLE
Finished

### PR DESCRIPTION
**Pull request by Alaa-Khalaila**

### CODE REVIEW

**Comment by Alaa-Khalaila** [comment]
image
image
image
image
image

File: Front-End/debug.log [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[1126/220453.041:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1126/225503.404:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (Besslan) [inline Front-End/debug.log]: You will need to add the `.log` files to `.gitignore` file

File: Front-End/src/App.css [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+.navbar{
+    height: 70px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 15px;
+    color: #54534F;
+    box-shadow: 1px 1px 5px ;
+    
+
```
Comment (Besslan) [inline Front-End/src/App.css]: remove the extra spaces

File: Front-End/src/App.css [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+}
+.logo{
```
Comment (Besslan) [inline Front-End/src/App.css]: add an empty line in  between the classes

File: Front-End/src/App.css [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+    height: 30px;
+    width: 100px;
+    margin-left: 18px;
+    margin-bottom: 8px;
+}
+.logo:hover{
+    cursor: pointer;
+}
+.search{
```
Comment (Besslan) [inline Front-End/src/App.css]: same as above

File: Front-End/src/App.css [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+    border: 1px solid ;
+    width: 60%;
+    height: 65%;
+    display: flex;
+    align-items: center;
+    border-radius: 40px;
+    padding-left: 15px;
+
```
Comment (Besslan) [inline Front-End/src/App.css:18]: remove the empty line

File: Front-End/src/App.css [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+}
+
+.search img{
+    height: 30px;
+    width: 30px;
+}
+.search input{
```
Comment (Besslan) [inline Front-End/src/App.css]: empty line in between

File: Front-End/src/App.css [hunk 7]
```diff
@@ -0,0 +1,296 @@
...
+    width: 90%;
+    height: 60%;
+    margin-left: 1px;
+    outline: none;
+    border: none;
+}
+
+
```
Comment (Besslan) [inline Front-End/src/App.css:44]: remove the extra line

File: Front-End/src/App.css [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+.log:hover{
+    color: black;
+    cursor: pointer;
+}
+.sign:hover{
+    background-color: #18294d;
+    cursor: pointer;
+}
+.log{
```
Comment (Besslan) [inline Front-End/src/App.css]: lines in between the classes

File: Front-End/src/App.css [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+    border: 1px solid #2896A9;
+    background-color: #2896A9;
+    color: white;
+    font-size: 15px;
+    font-weight: bold;
+}
+
+
+/* -------------------------------------------------- Interface -----------------------------------------*/
+
```
Comment (Besslan) [inline Front-End/src/App.css:93]: remove a line before and a line after the comment

File: Front-End/src/App.css [hunk 10]
```diff
@@ -0,0 +1,296 @@
...
+
+.wlc{
+    margin-top: 3px;
+
```
Comment (Besslan) [inline Front-End/src/App.css]: remove the empty line

File: Front-End/src/App.css [hunk 11]
```diff
@@ -0,0 +1,296 @@
...
+}
+
+.innerTxt{
+
```
Comment (Besslan) [inline Front-End/src/App.css]: remove the empty line

File: Front-End/src/App.css [hunk 12]
```diff
@@ -0,0 +1,296 @@
...
+.innerTxt{
+    
+    color:#54534F;
+    font-weight: bolder;
+    font-size: 25px;
+    padding:70px;
+    padding-left: 100px;
+
+}
```
Comment (Besslan) [inline Front-End/src/App.css]: remove the empty line and add a line after the class

File: Front-End/src/App.css [hunk 13]
```diff
@@ -0,0 +1,296 @@
...
+.innerTxt h4{
+    margin-top: 20px;
+    font-size: 20px;
+}
+.innerTxt button{
```
Comment (Besslan) [inline Front-End/src/App.css]: add a line in between

File: Front-End/src/App.css [hunk 14]
```diff
@@ -0,0 +1,296 @@
...
+    height: 60px;
+    width: 200px;
+    font-size: 25px;
+    color: #18294d;
+    background-color: white;
+    font-weight: bold;
+    border: none;
+    border-radius: 3px;
+}
+.innerTxt button:hover{
```
Comment (Besslan) [inline Front-End/src/App.css]: add a line in between

File: Front-End/src/App.css [hunk 15]
```diff
@@ -0,0 +1,296 @@
...
+    background-color: #18294d;
+    color: white;
+    cursor: pointer;
+}
+
+
```
Comment (Besslan) [inline Front-End/src/App.css:133]: remove a line before the comment

File: Front-End/src/App.css [hunk 16]
```diff
@@ -0,0 +1,296 @@
...
+    margin-left: 13%;
+    margin-top: 100px;
+    border-top: 1px solid silver;
+    border-bottom: 1px solid #54534F;
+    margin-bottom: 70px;
+    width: 73%;
+    color: #54534F;
+    height: 420px;
+}
+.goals h1{
```
Comment (Besslan) [inline Front-End/src/App.css]: add an empty  line in between

File: Front-End/src/App.css [hunk 17]
```diff
@@ -0,0 +1,296 @@
...
+
+.box{
+    width: 30%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 17px;
+    font-weight: bolder;
+}
+.box h2{
```
Comment (Besslan) [inline Front-End/src/App.css]: empty line in between

File: Front-End/src/App.css [hunk 18]
```diff
@@ -0,0 +1,296 @@
...
+    margin-bottom: 12px;
+    margin-top: -10px;
+    font-size: 23px;
+    letter-spacing: 2px;
+    
+}
+.box h4{
```
Comment (Besslan) [inline Front-End/src/App.css]: empty line in between

File: Front-End/src/App.css [hunk 19]
```diff
@@ -0,0 +1,296 @@
...
+
+.tt{
+    width: 87%;
+    margin-left: 7%;
+    margin-bottom: 15px;
+    font-size: 35px;
+    font-weight: 900;
+    color: #3C3B37;
+}
+.cards{
```
Comment (Besslan) [inline Front-End/src/App.css]: add an empty line in between

File: Front-End/src/App.css [hunk 20]
```diff
@@ -0,0 +1,296 @@
...
+    color: #3C3B37;
+    width: 85%;
+    margin-left: 7%;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    
+}
+.card{
```
Comment (Besslan) [inline Front-End/src/App.css]: remove the extra line and add an empty line in between

File: Front-End/src/App.css [hunk 21]
```diff
@@ -0,0 +1,296 @@
...
+    margin-bottom:10px;
+    margin-right: 2px;
+    border: 1px solid silver;
+    height: 480px;
+}
+.card:hover{
+    cursor: pointer;
+}
```
Comment (Besslan) [inline Front-End/src/App.css]: add lines in between the classes

File: Front-End/src/App.css [hunk 22]
```diff
@@ -0,0 +1,296 @@
...
+.card img{
+    height: 410px;
+    width: 400px;
+
```
Comment (Besslan) [inline Front-End/src/App.css]: remove the extra line

File: Front-End/src/App.css [hunk 23]
```diff
@@ -0,0 +1,296 @@
...
+/* ---------------------------------------------------- Become Instructor -----------------------*/
+.smile{
+    border-top: 1px #3C3B37 solid;
+    margin-top: 30px;
+    height: 600px;
+    display: flex;
+    align-items: center;
+    background-color: #c5e0ed;
+
```
Comment (Besslan) [inline Front-End/src/App.css]: same as above

File: Front-End/src/App.css [hunk 24]
```diff
@@ -0,0 +1,296 @@
...
+.smile{
+    border-top: 1px #3C3B37 solid;
+    margin-top: 30px;
+    height: 600px;
+    display: flex;
+    align-items: center;
+    background-color: #c5e0ed;
+    
+}
+.smile img{
```
Comment (Besslan) [inline Front-End/src/App.css]: add a line in between

File: Front-End/src/App.css [hunk 25]
```diff
@@ -0,0 +1,296 @@
...
+    height: 400px;
+    margin-left: 20%;
+
```
Comment (Besslan) [inline Front-End/src/App.css:232]: remove the extra empty line

File: Front-End/src/App.css [hunk 26]
```diff
@@ -0,0 +1,296 @@
...
+    color: white;
+    background-color: #2896A9;
+    border-radius: 5px;
+    border: none;
+}
+
+.smile button:hover{
+    cursor: pointer;
+    background-color: #18294d;
+
```
Comment (Besslan) [inline Front-End/src/App.css]: remove empty line

File: Front-End/src/App.css [hunk 27]
```diff
@@ -0,0 +1,296 @@
...
+/* ---------------------------------------- Team -----------------------------------------------*/
+
+.team{
+    margin-top: 70px;
+    display: flex;  
+    flex-direction: column;
+    align-items: center;
+    color: #3C3B37;
+}
+.team h1{
```
Comment (Besslan) [inline Front-End/src/App.css]: add an empty line in between the classes

File: Front-End/src/App.css [hunk 28]
```diff
@@ -0,0 +1,296 @@
...
+    font-size: 40px;
+    font-weight: bolder;
+    letter-spacing: 2px;
+}
+
+.members{
+    width: 65%;
+    display: flex;
+    justify-content: space-between;
+
```
Comment (Besslan) [inline Front-End/src/App.css:280]: remove the empty line

File: Front-End/src/App.css [hunk 29]
```diff
@@ -0,0 +1,296 @@
...
+    cursor: pointer;
+}
+
+.member{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 270px;
+    justify-content: space-evenly;
+}
```
Comment (Besslan) [inline Front-End/src/App.css]: add an empty line at the end

File: Front-End/src/components/Interface.js [hunk 30]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import wlc from "./pics/wlc.jpg"
+
+
+const wlcStyle={
+    backgroundImage:`url(${wlc})`,
+    height:"400px",
+    backgroundSize:"cover",
+    backgroundPosition:"center",
+}
```
Comment (Besslan) [inline Front-End/src/components/Interface.js]: why do you have styling here and not in the css file like the rest of the style? move it to the css file instead to be consistent

File: Front-End/src/components/Interface.js [hunk 31]
```diff
@@ -0,0 +1,28 @@
...
+    render() {
+        return (
+            <div style={wlcStyle} className="wlc">
+                <div className="innerTxt">
+                    <h1>For every student,<br></br>every classroom.<br></br>Real results.</h1>
+                    <h4>Build skills with courses, online from best instructors</h4>  
+                    <button>Join For Free</button> 
+                </div>
+                           
+
```
Comment (Besslan) [inline Front-End/src/components/Interface.js]: remove the empty lines

File: Front-End/src/components/Interface.js [hunk 32]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import wlc from "./pics/wlc.jpg"
+
+
+// const wlcStyle={
+//     backgroundImage:`url(${wlc})`,
+//     height:"400px",
+//     backgroundSize:"cover",
+//     backgroundPosition:"center",
+// }
```
Comment (Besslan) [inline Front-End/src/components/Interface.js]: delete  the commented lines

File: Front-End/debug.log (hunk 1 above)
Comment (Besslan) [reply Front-End/debug.log]: You haven't done this change yet

Comment (Alaa-Khalaila) [reply Front-End/debug.log]: what do you mean ?

**Pull request merged**